docs/notes/how-to-use-vineyard-operator.rst

```diff
@@ -624,7 +624,242 @@ above object as follows.
+    customresourcedefinition.apiextensions.k8s.io/vineyardds.k8s.v6d.io created
+    serviceaccount/vineyard-manager created
+    role.rbac.authorization.k8s.io/vineyard-leader-election-role created
+    clusterrole.rbac.authorization.k8s.io/vineyard-manager-role created
+    clusterrole.rbac.authorization.k8s.io/vineyard-metrics-reader created
+    clusterrole.rbac.authorization.k8s.io/vineyard-proxy-role created
+    clusterrole.rbac.authorization.k8s.io/vineyard-scheduler-plugin-role created
+    rolebinding.rbac.authorization.k8s.io/vineyard-leader-election-rolebinding created
+    clusterrolebinding.rbac.authorization.k8s.io/vineyard-kube-scheduler-rolebinding created
+    clusterrolebinding.rbac.authorization.k8s.io/vineyard-manager-rolebinding created
+    clusterrolebinding.rbac.authorization.k8s.io/vineyard-proxy-rolebinding created
+    clusterrolebinding.rbac.authorization.k8s.io/vineyard-scheduler-plugin-rolebinding created
+    clusterrolebinding.rbac.authorization.k8s.io/vineyard-scheduler-rolebinding created
+    clusterrolebinding.rbac.authorization.k8s.io/vineyard-volume-scheduler-rolebinding created
+    service/vineyard-controller-manager-metrics-service created
+    service/vineyard-webhook-service created
+    deployment.apps/vineyard-controller-manager created
+    certificate.cert-manager.io/vineyard-serving-cert created
+    issuer.cert-manager.io/vineyard-selfsigned-issuer created
+    mutatingwebhookconfiguration.admissionregistration.k8s.io/vineyard-mutating-webhook-configuration created
+    validatingwebhookconfiguration.admissionregistration.k8s.io/vineyard-validating-webhook-configuration created
+    make[1]: Leaving directory '/opt/caoye/v6d/k8s'
+    deployment.apps/vineyard-controller-manager condition met
+    Vineyard-Operator Ready
+    Installing vineyard cluster...
+    vineyardd.k8s.v6d.io/vineyardd-sample created
+    vineyardd.k8s.v6d.io/vineyardd-sample condition met
+    Vineyard cluster Ready
+
+.. raw:: html
+
+   </details>
+
+Check all vineyard pods are running.
+
+.. code:: bash
+
+    $ export KUBECONFIG=/tmp/e2e-k8s.config && kubectl get pod -n vineyard-system
+
+.. raw:: html
+
+   <details>
+   <summary><a>Expeced output</a></summary>
+
+.. code:: bash
+
+    NAME                                           READY   STATUS    RESTARTS   AGE
+    etcd0                                          1/1     Running   0          68s
+    etcd1                                          1/1     Running   0          68s
+    etcd2                                          1/1     Running   0          68s
+    vineyard-controller-manager-7f569b57c5-46tgq   2/2     Running   0          92s
+    vineyardd-sample-6ffcb96cbc-gs2v9              1/1     Running   0          67s
+    vineyardd-sample-6ffcb96cbc-n59gg              1/1     Running   0          67s
+    vineyardd-sample-6ffcb96cbc-xwpzd              1/1     Running   0          67s
+
+.. raw:: html
+
+   </details>
```

Comment:
Using `raw` html in both rst and markdown is quite hacky. We would try to implement the `.. details::` markup in upstream theme.

Reply:
Emmm, for `markdown`, there is a label like `<details>`. Do you mean to add a markup to [the repo](https://github.com/pradyunsg/furo)?

Reply:
Will implement inside vineyard itself.